docs/reference/publication_formats.md

```diff
@@ -1,12 +1,152 @@
+
+```
+{"id": "1"}
+{"id": "2"}
+```
+
 ## GeoJSON
 
-### Transformation specification
+```{admonition} Alpha consultation
+This section has [open issues](https://github.com/Open-Telecoms-Data/open-fibre-data-standard/issues?q=is%3Aopen+is%3Aissue+label%3A%22GeoJSON+format%22+).
+```
+
+This section describes how to publish data in GeoJSON format.
 
-To transform a JSON-format OFDS network to GeoJSON format, you must:
+If your data is small enough to fit into memory or if you are publishing data via an API, you should use the [small files and API responses option](#small-files-and-api-responses-option-1). If your data is too large to fit into memory, you should use the [streaming option](#streaming-option-1).
+
+### Small files and API responses option
+
+Publish separate GeoJSON [feature collections](https://datatracker.ietf.org/doc/html/rfc7946#section-3.3) for nodes and links, according to the [GeoJSON transformation specification](#geojson-transformation-specification).
+
+::::{tab-set}
+
+:::{tab-item} Nodes feature collection
+The following example shows a GeoJSON feature collection containing nodes:
+```{jsoninclude} ../../examples/geojson/nodes.geojson
+:jsonpointer:
+```
+:::
+
+:::{tab-item} Links feature collection
+The following example shows a GeoJSON feature collection containing links:
+```{jsoninclude} ../../examples/geojson/links.geojson
+:jsonpointer:
+```
+:::
+
+::::
+
+Each feature collection may contain features from one or more networks. The network each feature relates to is identified by its `.properties.network.id`.
+
+The following example shows a GeoJSON feature collection containing features from two networks:
+
+```{jsoninclude} ../../examples/geojson/multiple-networks.geojson
+:jsonpointer:
+```
+
+For data published via a paginated API, you should add a top-level `pages` object to the feature collection to provide links to the next and previous pages of results:
+
+::::{tab-set}
+
+:::{tab-item} Pages object schema
+```{jsonschema} ../../schema/network-package-schema.json
+:pointer: /properties/pages
+```
+:::
+
+:::{tab-item} API response example
+The following example shows a GeoJSON feature collection containing two features with links to the next and previous pages of results.
+```{jsoninclude} ../../examples/geojson/api-response.geojson
+:jsonpointer:
+```
+:::
+
+::::
+
+### Streaming option
+
+The streaming option describes how to publish multiple networks in GeoJSON format with support for streaming. You should only use this option if your data is too large to load into memory.
+
+The streaming option for GeoJSON is separate [Newline-delimited GeoJSON](https://stevage.github.io/ndgeojson/) files for nodes and links, in which each line is a GeoJSON feature structured according to the [GeoJSON transformation specification](#geojson-transformation-specification).
+
+The following example shows a newline-delimited GeoJSON file containing two features:
+
+```
+{"type": "Feature"}
```

Comment:
Can we add a few more example fields here too?

Reply:
Done in 2b4ae25476f61df5925ea849a54a3597d64ad6ab